migrations/1676731511125_initialize.cjs

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined
```

Comment:
What's this?

Reply:
https://github.com/salsita/node-pg-migrate/blob/9331f6fda98795e3f3733461f9b71345168b99d8/docs/migrations.md?plain=1#L3-L19